src/pages/Main/MainImg.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Autoplay, Pagination, Navigation } from 'swiper';
+import styled from 'styled-components';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import 'swiper/css/navigation';
+function MainImg({ mainImage }) {
+  return (
+    <div>
+      <Swiper
+        spaceBetween={30}
+        centeredSlides={true}
+        autoplay={{
+          delay: 2500,
+          disableOnInteraction: false,
+        }}
+        pagination={{
+          clickable: true,
+        }}
+        navigation={true}
+        modules={[Autoplay, Pagination, Navigation]}
+      >
+        <SwiperSlide>
+          <SlideImg src={mainImage[0]?.url} />
+        </SwiperSlide>
+        <SwiperSlide>
+          <SlideImg src={mainImage[1]?.url} />
+        </SwiperSlide>
+        <SwiperSlide>
+          <SlideImg src={mainImage[2]?.url} />
+        </SwiperSlide>
+        <SwiperSlide>
+          <SlideImg src={mainImage[3]?.url} />
+        </SwiperSlide>
+        <SwiperSlide>
+          <SlideImg src={mainImage[4]?.url} />
+        </SwiperSlide>
+        <SwiperSlide>
+          <SlideImg src={mainImage[5]?.url} />
+        </SwiperSlide>
+      </Swiper>
```

Comment:
```suggestion
 {mainImage.map(list => {
```

반복되는 구조인것 같은데 mainImage가 배열로 이루어져있네요!
이 부분 map 메서드를 사용하면 더욱 간결하게 구현이 가능합니다!